src/anari/include/anari/anari_cpp/Traits.h

```diff
@@ -44,6 +44,11 @@ ANARI_TYPEFOR_SPECIALIZATION_STRING(char[], ANARI_STRING);
 ANARI_TYPEFOR_SPECIALIZATION_STRING(const char *, ANARI_STRING);
 ANARI_TYPEFOR_SPECIALIZATION_STRING(const char[], ANARI_STRING);
 
+ANARI_TYPEFOR_SPECIALIZATION(char *[], ANARI_STRING_LIST);
+ANARI_TYPEFOR_SPECIALIZATION(char **, ANARI_STRING_LIST);
+
+ANARI_TYPEFOR_SPECIALIZATION(bool, ANARI_BOOL);
```

Comment:
We explicitly don't specialize on `bool` because there is no guarantee that `sizeof(bool) == anari::sizeOf(ANARI_BOOL)`. We _do_ overload/specialize on `bool` where appropriate in libraries like `helium`, but we don't want to do this for the actual C++ type.

Reply:
Makes sense, I completely missed that. I'll drop that commit.